README.md

```diff
@@ -43,6 +43,9 @@ through plugins.
   Kubernetes.
 - **Dynamic Load Balancing**: Load balance traffic across multiple backend
   services.
+- **Hash-based Load Balancing**: Load balance consisten hashing/sticky sessions.
+- **Circuit-Breaking**: Smarter trackling of unhealthy upstream instances.
+- **Health Checks:** Passive and Active monitoring of your services.
```

Comment:
Any reason for "Active" to be capitalized here?

Reply:
@jbergstroem will fix thanks